packages/react-server/src/ReactFlightServer.js

```diff
@@ -204,7 +204,18 @@ function findCalledFunctionNameFromStackTrace(
     const callsite = stack[i];
     const functionName = callsite[0];
     const url = devirtualizeURL(callsite[1]);
-    if (filterStackFrame(url, functionName)) {
+    const lineNumber = callsite[2];
+    const columnNumber = callsite[3];
+    const enclosingLineNumber = callsite[4];
+    const enclosingColumnNumber = callsite[5];
+    if (
+      filterStackFrame(
+        url,
+        functionName,
+        enclosingLineNumber || lineNumber,
+        enclosingColumnNumber || columnNumber,
```

Comment:
Do we ever care about the enclosing line number? We assume that they're always in the same module so can't we only pass the non-enclosing always?

If we did care about the difference, then it's probably not enough to assume it's the enclosing always.

Reply:
Just passing in the location of the callsite. Enclosing location is only relevant for name mapping which you probably shouldn't do here.